<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_buy">
     <attribute name="title">
      <string>장보기</string>
     </attribute>
     <widget class="QListWidget" name="list_select_cook">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>340</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="list_meet">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>190</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QListWidget" name="list_vegetable">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>270</y>
        <width>190</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QListWidget" name="list_mushroom">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>480</y>
        <width>190</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cook">
     <attribute name="title">
      <string>요리 등록</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_cookname">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>요리 이름</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_meet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="list_meet_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>270</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QListWidget" name="list_vegetable_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>270</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QListWidget" name="list_mushroom_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>680</y>
        <width>270</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QDial" name="dial_meet">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDial" name="dial_vegetable">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>490</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_vegetable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_mushroom">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>760</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mushroom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>760</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_meet">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_vegetable">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mushroom">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>760</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_meet">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0 g</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_vegetable">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>500</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0 개</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_mushroom">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>770</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0 개</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_registor">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>900</y>
        <width>81</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
